src/bundle/Resources/public/js/scripts/helpers/context.helper.js

```diff
@@ -1,16 +1,17 @@
 let { bootstrap, flatpickr, moment, Popper, Routing, Translator } = window;
 let adminUiConfig = window.ibexa?.adminUiConfig;
+let rootNode = document.body;
```

Comment:
I think there suppose to be window since previously rootNode was window

Reply:
For standalone version this is node wrapper of Assets Library Widget not window. Michał sugested `rootDOMElement`